src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h

```diff
@@ -64,6 +64,7 @@ namespace OpenMS
     double pmass;
```

Comment:
I think these structs should at least have some //< comments for the individual fields